eng/.docsettings.yml

```diff
@@ -180,6 +180,7 @@ known_content_issues:
   - ['sdk/textanalytics/azure-ai-textanalytics/swagger/README.md', '#3113']
   - ['sdk/translation/azure-ai-documenttranslator/swagger/README.md', '#3113']
   - ['sdk/webpubsub/azure-messaging-webpubsub/swagger/README.md', '#3113']
+  - ['sdk/confidentialledger/azure-data-confidentialledger/swagger/README.md', '#3113']
```

Comment:
nit:  this should be alphabetically ordered

Reply:
fixed